content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1222,20 +1222,20 @@ const deleteUser = await prisma.user.delete({
 
 `createMany` creates multiple records in a transaction.
 
-#### Options
+**Options**
```

Comment:
This is problematic, as it will change the anchor of all the other anchors of the "Options" headlines.

Why should this not be a headline any more?

(Also applies for "Return Type", Remarks and so on below)